### DISCUSSION TITLE
copy array operation functions from cooler.util to bioframe.arrops

### DISCUSSION THREAD

**Discussion by golobor** [open2c/bioframe#149] [General]
This includes rlencode/rldecode etc...
Also, consider adding code from this repo: https://gist.github.com/nvictus/66627b580c13068589957d6ab0919e66

**Comment by nvictus** [comment]
btw, it was all migrated to cooltools.lib.runlength... we could make bioframe its new home

**Comment by golobor** [comment]
Good! I'll copy it then to implement genomecov(). I won't remove it from
cooltools at least until we release bioframe

**Comment by gfudenberg** [comment]
any further thoughts on this migration? @golobor @nvictus
